gateways/checkoutcom/request_builder.go

```diff
@@ -35,7 +41,35 @@ func buildChargeParams(authRequest *sleet.AuthorizationRequest) (*payments.Reque
 			Email: common.SafeStr(authRequest.BillingAddress.Email),
 			Name:  authRequest.CreditCard.FirstName + " " + authRequest.CreditCard.LastName,
 		},
-	}, nil
+	}
+
+	// see documentation for instructions on stored credentials, merchant-initiated transactions, and subscriptions:
+	// https://www.checkout.com/docs/four/payments/accept-payments/use-saved-details/about-stored-card-details
+	if authRequest.ProcessingInitiator != nil {
```

Comment:
Would it make sense to move the processor initiator code into its own function?

Reply:
Agreed